teslajsonpy/car.py

```diff
@@ -950,3 +962,41 @@ async def unlock(self) -> None:
         if data and data["response"]["result"] is True:
             params = {"locked": False}
             self._vehicle_data["vehicle_state"].update(params)
+
+    async def vent_windows(self) -> None:
+        """Vent Windows."""
+        data = await self._send_command(
+            "WINDOW_CONTROL",
+            path_vars={"vehicle_id": self.id},
+            command="vent",
+            lat=0,
+            long=0,
+            wake_if_asleep=True,
+        )
+        if data and data["response"]["result"] is True:
+            params = {
+                "fd_window": 1,
```

Comment:
I was actually curious about this - does the Tesla API JSON response actually return 1? Or is it 255 for a fully open window and anything between 0-255 for a partially open window? That seems to be the case with the trunk.